english/love/love.md

```diff
@@ -16,3 +16,5 @@ cannot make you understand the feeling of it. Anyways, [here's
 something comepletely different.](../marshmallow.md)
 
 If all else fails, you could [try this.](../read-a-book/read-a-book.md)
+
+Love feels eternal but the pain it adds to your heart is painful [The Pain] (https://www.quora.com/What-is-it-like-when-you-feel-the-pain-of-loving-someone-when-its-one-sided) to see how people manage the pain
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.